src/android/patches/makeSettingsPatch.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 module.exports = function makeSettingsPatch(name, dependencyConfig, projectConfig) {
   const relative = path.relative(
-    path.dirname(projectConfig.settingsGradlePath),
+    projectConfig.settingsGradlePath,
```

Comment:
That has to be dirname, because path.relative given the path to file, will generate wrong relative path.